examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>doc type references should use paths, and they can use paths from the home directory of the documentation. As here we have the <code>conf.py</code>  within the examples directory, this is set as the "home" of the docs. You can always double check the references with:</p><p> </p><pre class="ql-syntax" spellcheck="false">sphobjinv suggest -t 90 -u https://pymc-examples.readthedocs.io/en/latest/ posterior_predictive
&nbsp;
No inventory at provided URL.
Attempting "https://pymc-examples.readthedocs.io/en/latest/objects.inv" ...
Remote inventory found.
&nbsp;
:std:doc:<code>diagnostics_and_criticism/posterior_predictive</code>
</pre><p> </p><p>so here, using </p><p> </p><pre class="ql-syntax" spellcheck="false">{doc}<code>diagnostics_and_criticism/posterior_predictive</code>
</pre><p> </p><p>should work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>